lib/cn/csched_sp3.c

```diff
@@ -2662,6 +2684,44 @@ sp3_destroy(struct csched *handle)
 
     rest_server_remove_endpoint(ENDPOINT_FMT_KVDB_CSCED, sp->kvdb_alias);
 
+    if (1) {
+        struct rusage ru;
+
+        log_info("%8s %6s %12s %12s %7s %7s %7s %7s",
+                 "rule", "jobs",  "keysin", "keysout", "kbr", "kbw", "vbr", "vbw");
+
+        for (enum cn_rule r = CN_RULE_NONE; r < CN_RULE_MAX; ++r) {
+            const struct cn_merge_stats *stats = &sp->sp_mstatsv[CN_RULE_MAX - 1 - r];
+
+            if (stats->ms_jobs == 0)
+                continue;
+
+            log_info("%8s %6u %12lu %12lu %7lu %7lu %7lu %7lu",
+                     (stats == sp->sp_mstatsv) ? "total" : cn_rule2str(stats - sp->sp_mstatsv),
+                     stats->ms_jobs,
+                     stats->ms_keys_in, stats->ms_keys_out,
+                     stats->ms_kblk_read.op_size >> 20,
+                     stats->ms_kblk_write.op_size >> 20,
+                     (stats->ms_vblk_read1.op_size +
+                      stats->ms_vblk_read2.op_size) >> 20,
+                     stats->ms_vblk_write.op_size >> 20);
+
+            cn_merge_stats_add(&sp->sp_mstatsv[CN_RULE_NONE], stats);
+        }
+
+        getrusage(RUSAGE_SELF, &ru);
+
+        ru.ru_minflt -= sp->sp_rusage.ru_minflt;
+        ru.ru_majflt -= sp->sp_rusage.ru_majflt;
+        ru.ru_nswap -= sp->sp_rusage.ru_nswap;
+        ru.ru_inblock -= sp->sp_rusage.ru_inblock;
+        ru.ru_oublock -= sp->sp_rusage.ru_inblock;
+
+        log_info("minflt %lu, majflt %lu, nswap %lu, inbytes %lu, outbytes %lu",
+                 ru.ru_minflt, ru.ru_majflt,ru.ru_nswap,
+                 ru.ru_inblock * 512, ru.ru_oublock * 512);
+    }
+
```

Comment:
This is temporary until we wire up a REST hook to get the information on-demand.